README.md

```diff
@@ -558,6 +558,17 @@ and parsed, and its properties are set as arguments. If present, the
 `description` parameter customizes the description of the config (`key`) option
 in the usage string.
 
+An optional `parseFn` argument can be passed to provide a custom config
+parser:
```

Comment:
Should we perhaps mention that the function should be synchronous?

Or maybe just describe the expected API/contract of the parser function? E.g.

> The parser function should accept a file path argument and synchronously return an object whose keys will be applied to `argv`.